account_asset_comment/migrations/14.0.1.0.0/pre-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 ForgeFlow S.L.  <https://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+def comment_to_note(cr):
+    cr.execute(
+        """
+        UPDATE account_asset
+        SET note = CASE WHEN note IS NULL OR note = '' THEN comment
+            WHEN comment IS NOT NULL THEN COALESCE(note, '') || comment
+            ELSE note END"""
+    )
+
+
+def migrate(cr, version):
+    comment_to_note(cr)
```

Comment:
@MiquelRForgeFlow After the migration from "comment" to "note", the old "comment" field should be removed.